Seminars/Sem.08/Pract.08/Task01/Student.cpp

```diff
@@ -0,0 +1,83 @@
+#include "Student.h"
+
+#include <cstring>
+#pragma warning(disable : 4996)
+
+#include <iostream>
+
+Student::Student(const char* name, unsigned FN) {
+	setName(name);
+	setFN(FN);
+}
+
+const Grade* Student::getGrades() const {
+	return _grades;
+}
+
+unsigned Student::getNumberOfGrades() const {
+	return _numberOfGrades;
+}
+
+unsigned Student::getFN() const {
+	return _FN;
+}
+
+const MyString& Student::getName() const {
+	return _name;
+}
+
+void Student::setName(const char* name) {
+	_name = MyString(name);
+}
+
+void Student::setGradeAtIndex(size_t ind, double newValue) {
```

Comment:
Maybe it would be better if this function is bool to check whether you have failed to set grade